sequencer/src/block/full_payload/payload.rs

```diff
@@ -0,0 +1,233 @@
+use crate::{
+    block::{
+        full_payload::ns_table::{NsIndex, NsTable, NsTableBuilder},
+        namespace_payload::{Index, Iter, NsPayload, NsPayloadBuilder, NsPayloadRange, TxProof},
+    },
+    NamespaceId, NodeState, Transaction,
+};
+use hotshot_query_service::availability::QueryablePayload;
+use hotshot_types::{
+    traits::{BlockPayload, EncodeBytes},
+    utils::BuilderCommitment,
+    vid::{VidCommon, VidSchemeType},
+};
+use jf_vid::VidScheme;
+use serde::{Deserialize, Serialize};
+use sha2::Digest;
+use std::{collections::HashMap, fmt::Display, sync::Arc};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct Payload {
+    // Concatenated payload bytes for each namespace
+    #[serde(with = "base64_bytes")]
+    payload: Vec<u8>,
```

Comment:
Would be nice to add some comments following this helpful document: https://www.notion.so/espressosys/Block-payload-format-with-namespace-transaction-tables-81425c5bd3ba495bafc3198ce287c487

Reply:
addressed in https://github.com/EspressoSystems/espresso-sequencer/pull/1499/commits/5c9fe51fa9f95bb392d5c8446166068663ef0ac1. Added a detailed spec in rustdoc